src/main/java/frc/robot/commandgroups/bits/TestBallFlow.java

```diff
@@ -24,14 +20,6 @@ public TestBallFlow(Hood hood, Flap flap, Conveyor conveyor, Intake intake, Shoo
         this.intake = intake;
         this.shooter = shooter;
         this.conveyor = conveyor;
-        ShootCargo shootCargo = new ShootCargo(shooter, hood, conveyor, flap, Constants.Conveyor.DEFAULT_POWER::get);
-        PickUpCargo pickUpCargo = new PickUpCargo(conveyor, flap, intake, Constants.Conveyor.DEFAULT_POWER.get(), () -> 0.5);
-
-        addCommands(
-                pickUpCargo.withInterrupt(pickUpCargoIsFinished),
-                new WaitCommand(5),
-                shootCargo.withInterrupt(shootCargoIsFinished)
-        );
```

Comment:
Why did you remove the whole point of this command?

Reply:
I commented what gave me errors, what's important can be brought back